plugins/redis-cli/_redis-cli

```diff
@@ -17,9 +17,9 @@ _1st_arguments=(
   # 'config resetstat: reset the stats returned by INFO'
   'dbsize:return the number of keys in the selected database'
   # 'debug object:get debugging information about a key'
-  # 'debug setgfault:make the server crash'
+  # 'debug segfault:make the server crash'
```

Comment:
Not certain what it's doing; couldn't find a sign of `setgfault`, and it's a comment, so, it probably doesn't matter.